variables.tf

```diff
@@ -65,5 +60,22 @@ variable "plan_selection_tag" {
       value = ""
     }
   ]
+}
 
-}
\ No newline at end of file
+variable "rules" {
+  description = "List of rules to attach to the plan"
+  type        = list(map)
+  default = [
+    {
+      name              = "daily"
+      schedule          = "cron(0 12 * * ? *)"
+      continuous_backup = true
+      vault             = "Backup"
```

Comment:
seems you do not use `continuous_backup` and `vault` params,
as I understood it is supposed they will replace var.enable_continuous_backup and local.vault_name ones ?

same for recovery_point_tags param one, and in code you use
```terraform
recovery_point_tags = {
      Environment = var.env
    }
```

Reply:
the aws_backup_plan.name its  rule name and aws_backup_selection resource block names and name params all have `daily` hardcoded naming, can we maybe have this namings more general/configurable?